lib/pharos/config_schema.rb

```diff
@@ -94,6 +94,7 @@ def self.messages
         optional(:name).filled(:str?)
         optional(:api).schema do
           optional(:endpoint).filled(:str?)
+          optional(:feature_gates).filled
```

Comment:
IMO this should be moved under `control_plane` key as these feature gates will now be set on all control plane components.

Reply:
Maybe the best solution is to define config options for all components.

Reply:
If we can safely propagate all the control plane feature gates to all control plane components, assuming that a component will just ignore a feature it does not understand, I think having them defined on `control_plane` level makes users lifes so much easier.

Reply:
oh, we already have `control_plane` option

Reply:
moved api feature gates under `control_plane`